src/deepsparse/transformers/utils/storage_kv_cache.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+
+_LOGGER = logging.getLogger(__name__)
+
+__all__ = ["SessionStorageKVCache"]
+
+
+class SessionStorageKVCache(dict):
```

Comment:
do we want to override any default methods to include more validation? additionally, we may want to add a capacity/memory to the dict eventually (ie LRU caching) might want to think a bit more on if this is the way to go

Reply:
The new design simplifies so many things that I would not do any (over)engineering here. So far, all we need from this object is the ability to assign key-value pairs and fetch a value given a key. In the future, we might expand this class, but for now, I do not see a good reason.